recipes/isl/build.sh

```diff
@@ -0,0 +1,10 @@
+if [ "$(uname)" = 'Darwin' ]
+then
+    export CC="clang"
+    export CXX="clang++"
+fi
+
+./configure --prefix="$PREFIX" --with-gmp-prefix="$PREFIX"
+make
+make check
```

Comment:
Change this to match this [line]( https://github.com/conda-forge/vigra-feedstock/blob/master/recipe/build.sh#L78 ) uncommented.